kiali/kiali
# GRPC Route wizard in Service Actions

### Describe the change

In Service details Action menu, added "Create/Update K8s GRPC Routing" items.
Updated "K8s Gateway API Routing" to "K8s HTTP Routing".
Added 'GRPC Routing' wizard.

![Screenshot from 2024-06-11 18-36-52](https://github.com/kiali/kiali/assets/604313/58d6e0dc-e3ea-4072-92e8-f7a6916caf02)


GRPCRoute object has validations and references.

![Screenshot from 2024-06-11 18-38-53](https://github.com/kiali/kiali/assets/604313/6b5e719d-4f8e-41c0-9899-849e99ab84cc)

![Screenshot from 2024-06-11 18-37-18](https://github.com/kiali/kiali/assets/604313/8cbfd4bb-4a56-4ad7-91e9-d4ad26b7366b)

Service list with gRPC route configs:
![Screenshot from 2024-06-11 18-37-06](https://github.com/kiali/kiali/assets/604313/475d873f-c4d7-45e9-a25c-70833819794d)




### Steps to test the PR

Check the functionality of Service wizard "K8s GRPC Routing" in create/update/delete.
Verify that GRPCRoute has validations, 'KIA1402' and 'KIA1401', like in HTTPRoute.

### Automation testing

added e2e, unit and cypress tests

### Issue reference
https://github.com/kiali/kiali/issues/7223
